python/ray/tests/test_client.py

```diff
@@ -479,16 +479,11 @@ def test_serializing_exceptions(ray_start_regular_shared):
 def test_invalid_task(ray_start_regular_shared):
     with ray_start_client_server() as ray:
 
-        @ray.remote(runtime_env="invalid value")
-        def f():
-            return 1
-
-        # No exception on making the remote call.
-        ref = f.remote()
+        with pytest.raises(TypeError):
 
-        # Exception during scheduling will be raised on ray.get()
-        with pytest.raises(Exception):
-            ray.get(ref)
+            @ray.remote(runtime_env="invalid value")
```

Comment:
now we raise the error earlier, so you expect an error before `ray.get()`. this helps catching errors when defining the remote function.